cygwin.rst

```diff
@@ -2,119 +2,134 @@
 Working with Docs in Windows using Cygwin
 ******************************************
 
+.. _cygwin-python:
+
+Python 3
+-------------------------------
+
+You need to install `Python 3 <https://www.python.org/downloads/>`_.For instructions, see `this video <https://www.youtube.com/watch?v=oHOiqFs_x8Y>`_. Make sure to select the option "Add python to the Path", otherwise you will need to add it manually.
+
 Install Cygwin
 -------------------------------
 
-Cygwin is a Windows implementation of many GNU/Linux commands usable from \*NIX command prompt. Install `cygwin <https://cygwin.com/install.html/>`_ and add its path to Windows, `for instructions <https://www.howtogeek.com/howto/41382/how-to-use-linux-commands-in-windows-with-cygwin/>`_.
+Cygwin is a Windows implementation of many GNU/Linux commands usable from \*NIX command prompt. Download `Cygwin <https://www.cygwin.com/install.html/>`_. Once you download the setup file, refer to these `instructions <https://www.davidbaumgold.com/tutorials/set-up-python-windows/>`_.
```

Comment:
Not really. Because to use pip commands python is needed which can be installed following the steps in the video you mentioned but cygwin most recognize it in most cases unless forced to so using the commands mentioned in the tutorial.